### DISCUSSION TITLE
I cant get '.' to work???

### DISCUSSION THREAD

**Discussion by brittGit** [lcn2/calc#73] [Q&A] [answered]
I am trying to sum a LOT of numbers and using the . feature would be a big help but it doesnt work for me?

I get the answer to a previous sum then I write "calc . + 26.10" and it gives me the result of "26.1" instead of adding up the previous sum plus 26.10....what am I doing wrong? Does the dot feature not work anymore?

Kindly,
Brittany

**Comment by lcn2** [comment, marked as answer]
> I am trying to sum a LOT of numbers and using the . feature would be a big help but it doesnt work for me?
> 
> I get the answer to a previous sum then I write "calc . + 26.10" and it gives me the result of "26.1" instead of adding up the previous sum plus 26.10....what am I doing wrong? Does the dot feature not work anymore?
> 
> Kindly, Brittany

When calc starts up, it starts with the value of 0.

So:

```calc
calc
C-style arbitrary precision calculator (version 2.14.1.5)
Calc is open software. For license details type:  help copyright
[Type "exit" to exit, or "help" for help.]

; . + 26.1
	26.1
; . + 26.1
	52.2
; . + 26.1
	78.3
```

works because the value "0" has "26.1" added to it again and again increasing.

If however, you were to have calc exit and restart:

```calc
$ calc
C-style arbitrary precision calculator (version 2.14.1.5)
Calc is open software. For license details type:  help copyright
[Type "exit" to exit, or "help" for help.]

; . + 26.1
	26.1
; exit
$ calc
C-style arbitrary precision calculator (version 2.14.1.5)
Calc is open software. For license details type:  help copyright
[Type "exit" to exit, or "help" for help.]

; . + 26.1
        26.1
; exit
```

Then on each calc restart, 26.1 is added to the initial value of "0".

This is the same thing:

```sh
$ : rubin; calc . + 26.10
	26.1
$ calc . + 26.10
	26.1
$ calc . + 26.10
	26.1
```

In other words, each time time calc restarts with the value of 0, 26.10 is added tot producing a value of 23.1.

**Reply by brittGit** [reply]
Thank you! You also just helped me learn that I was using calc wrong in the first place....I was typing "calc 1 + 1" EVERY time I wanted to use the calc function...instead of just typing "calc" and hitting enter to get into the program....I tried this again while being in calc and it worked. My bad and thank you!!!

**Comment by lcn2** [comment]
You are most welcome, @brittGit
